api/src/handlers/utils/errors.js

```diff
@@ -0,0 +1,5 @@
+export const AuthorizationError = (message, code = 403) => {
```

Comment:
IMO `AuthorizationError` is always associated with 403, would be nice to have an `HttpError` extending default `Error` with a `statusCode` and an `AuthorizationError` extending `HttpError` setting the statusCode ?
See https://github.com/marmelab/admin-on-rest/blob/master/src/util/HttpError.js